kcp/kcp-e2e/setup-ws-e2e.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+set -ex
+
+source ./kcp/utils.sh
```

Comment:
These scripts appear to assume that they are run from a different directory then they are defined in. We should probably use something like `SCRIPTPATH` (grep the codebase for instances of this), so that the script can get the path of itself, and use that to determine the location of utils.sh. 

This way, the scripts can be run standalone via e.g. `./setup-e2-e2e.sh`